build/config/compiler/BUILD.gn

```diff
@@ -882,7 +882,7 @@ config("optimize") {
     cflags = [ "/O1" ] + common_optimize_on_cflags + [ "/Oi" ]
   } else if (is_ios) {
     cflags = [ "-Os" ] + common_optimize_on_cflags  # Favor size over speed.
-  } else if (is_android || is_fuchsia || is_linux) {
+  } else if (is_android || is_fuchsia) {
```

Comment:
I'd appreciate if you amended the commit message to indicate that this was a revert of https://github.com/flutter/buildroot/commit/a4f3c4d5023e080ee50596e6623d179e9c5f839b for future commit archaeologists.

Reply:
It's not actually a revert - I'm only dropping the `is_linux`, but I'll mention that commit in the commit message.